tests/integration/study_data_blueprint/test_link.py

```diff
@@ -0,0 +1,294 @@
+            "hurdles-cost": False,
+            "link-style": "plain",
+            "link-width": 1.0,
+            "loop-flow": False,
+            "transmission-capacities": "enabled",
+            "use-phase-shifter": False,
+        }
+        assert expected == res.json()
+
+        # Test update link same area
+
+        res = client.put(
+            f"/v1/studies/{study_id}/links",
+            json={
+                "area1": area1_id,
+                "area2": area1_id,
+                "hurdles-cost": False,
+            },
+        )
+        assert res.status_code == 422
+        expected = {"description": "Area 1 and Area 2 can not be the same", "exception": "LinkValidationError"}
+        assert expected == res.json()
+
+        # Test update link with non existing area
+
+        res = client.put(
+            f"/v1/studies/{study_id}/links",
+            json={
+                "area1": area1_id,
+                "area2": "id_do_not_exist",
+                "hurdles-cost": False,
+            },
+        )
+        assert res.status_code == 422
+        expected = {
+            "description": "The link area 1 -> id_do_not_exist is not present in the study",
+            "exception": "LinkValidationError",
+        }
+        assert expected == res.json()
+
+        # Test update link fails when given wrong parameters
+        if study_type == "raw":
+            res = client.post(
+                f"/v1/studies/{study_id}/commands",
+                json=[
+                    {
+                        "action": "update_link",
+                        "args": {
+                            "area1": area1_id,
+                            "area2": area2_id,
+                            "parameters": {"hurdles-cost": False, "wrong": "parameter"},
+                        },
+                    }
+                ],
+            )
+            assert res.status_code == 500
+            expected = {
+                "description": "Unexpected exception occurred when trying to apply command CommandName.UPDATE_LINK: 422: One or more fields are forbidden",
+                "exception": "CommandApplicationError",
+            }
+            assert expected == res.json()
+
+        # Test update link variant returns only modified values
+
+        if study_type == "variant":
+            res = client.put(
+                f"/v1/studies/{study_id}/links",
+                json={
+                    "area1": area1_id,
+                    "area2": area2_id,
+                    "hurdles-cost": False,
+                },
+            )
+            assert res.status_code == 200
+
+            res = client.get(f"/v1/studies/{study_id}/commands")
+            commands = res.json()
+            command_args = commands[-1]["args"]
+            assert command_args["parameters"]["hurdles-cost"] == False
+            assert "loop-flow" not in command_args["parameters"]
```

Comment:
You could be more strict and check that the content equals exactly {"parameters": {"hurdles-cost": False}}